posthog/models/person.py

```diff
@@ -50,6 +50,19 @@ def merge_people(self, people_to_merge: List["Person"]):
 
             capture_internal(event, self.distinct_ids[-1], None, None, now, now, self.team.id)
 
+    def split_person(self, main_distinct_id: str):
+        with transaction.atomic():
+            distinct_ids = Person.objects.get(pk=self.pk).distinct_ids
+            if not main_distinct_id:
+                self.properties = {}
```

Comment:
Not sure why we clear the properties if `main_distinct_id` is not passed

Reply:
I have tried to make this clear in the interface (see screenshot above). Maybe I need to change the wording?